Recorded future Feed Integration (#5463)
demisto/content: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture_description.md

## Recorded Future Feed

> This integration downloads from Recorded Future a list of IP Addresses, domains, urls, or file hash with known risk. 

Reviewer: ```suggestion
This integration downloads from Recorded Future a list of IP addresses, domains, URLs, or file hashes with known risk associations. 
```

The risk list includes risk scores and the supporting evidence details.

The feed can be configured to load a specific risk list by specifying a risk rule. 

> If no risklist is specified, Recorded Futures default risklist is used. 

Reviewer: ```suggestion
If no risk list is specified, Recorded Future's default risk list is used. 
```

> The feed can also be configured to load a custom risklist from a user-specified Recorded Future file path.

Reviewer: ```suggestion
The feed can also be configured to load a custom risk list from a user-specified Recorded Future file path.
```

Reviewer: `The feed`? Which feed? Please specify.

> If no file path is specified, the default risklist file is used. 

Reviewer: ```suggestion
If no file path is specified, the default risk list file is used. 
```

> To access this resource, you need a valid Recorded Future API token.

Reviewer: ```suggestion
To access this resource, a valid Recorded Future API token is required.
```
